app/api/api_v1/endpoints/projects.py

```diff
@@ -76,13 +76,23 @@ def create_project(
     return crud.project.create(db=db, obj_in=project_in)
 
 
-@router.get("/default_config", response_model=schemas.TestEnvironmentConfig)
-def default_config() -> schemas.TestEnvironmentConfig:
+@router.get(
+    "/default_config", response_model=Union[dict, schemas.TestEnvironmentConfig]
+)
+def default_config() -> Union[dict, schemas.TestEnvironmentConfig]:
     """Return default configuration for projects.
 
     Returns:
         List[Project]: List of projects
     """
+    if not default_environment_config:
+        return {
+            "alert": "No program configuration file was found. "
+            "If you want default values for the project configuration, please, "
+            "create a default_project.config file inside "
```

Comment:
Can we use the TEST_ENVIRONMENT_CONFIG_NAME var from app/default_environment_config.py ?

Reply:
In fact No, default_environment_config is related to the instance of TestEnvironmentConfig, and TEST_ENVIRONMENT_CONFIG_NAME refers to the file name.

Reply:
I think I expressed myself poorly. What I thought of was replacing the file name in the text 'default_project.config' with the variable/count 'TEST_ENVIRONMENT_CONFIG_NAME'.

This way we will avoid future problems if we change the name of this file.